Resources/AzureDevOps/JobTemplates/deploy-environment.yml

```diff
@@ -0,0 +1,230 @@
+parameters:
+  AzureSubscription: 'SFA-CDH-Dev/Test (962cae10-2950-412a-93e3-d8ae92b17896)'  
+  SetSubscriptionScript: 'https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/PSScripts/Set-EsfaResourceGroupTags.ps1'  
```

Comment:
This param name should match the name of the script, eg SetEsfaResourceGroupTagsScript